web/massastation/src/models/DomainModel.ts

```diff
@@ -4,4 +4,5 @@ export interface DomainModel {
   description: string;
   metadata: string;
   icon: JSX.Element | null;
```

Comment:
```suggestion
```

You can remove this var.
I created it in the past to store the icon, but you've added the `favicon`.